jenkins_jobs/modules/triggers.py

```diff
@@ -1266,6 +1266,9 @@ def github_pull_request(registry, xml_parent, data):
             False,
         ),
     ]
+    cron_string = data.get("cron", "") if not None else ""
```

Comment:
`if not None`? this is always `True`, so you always get the value of `data.get("cron", "")`, the `else part is dead code